app/retail/templates/settings/email.html

```diff
@@ -41,13 +52,35 @@ <h5>{% trans "Basic Preferences" %}</h5>
         </div>
       </div>
       {% csrf_token %}
-      <input class='button button--primary' type='submit' name='submit' value="Go">
+      <input class='button button--primary' type='submit' name='submit' value="Save">
     </form>
   </div>
 {% endblock %}
 
 {% block scripts %}
 <script>
+    $("document").ready(function(){
```

Comment:
I think `$(document)` (the gobal variable) instead of `$("document")` (the string) ?